Modules/Filtering/DistanceMap/include/itkDirectedHausdorffDistanceImageFilter.h

```diff
@@ -185,7 +185,7 @@ class ITK_TEMPLATE_EXPORT DirectedHausdorffDistanceImageFilter : public ImageToI
   DistanceMapPointer m_DistanceMap;
 
   Array<RealType>       m_MaxDistance;
-  Array<IdentifierType> m_PixelCount;
+  Array<IdentifierType> m_PixelCount = Array<RealType>::Fill(NumericTraits<IdentifierType>::ZeroValue());
```

Comment:
Maybe `Filled` instead of `Fill`?

Reply:
Thanks for giving this a thought. `Filled` is not a member of `itk::Array`, only one of `itk::FixedArray`.

Reply:
What do you think about a simple `{}` initialization?

    Array<IdentifierType> m_PixelCount{};

Reply:
Looks cleanest to me. Done in afee6cc; I confess that given that I do not use ITK on a regular basis I miss the best elegant ways to initialize some of ITK fundamental types.